src/SourceBuild/content/test/Microsoft.DotNet.UnifiedBuild.Tests/Microsoft.DotNet.UnifiedBuild.Tests.csproj

```diff
@@ -0,0 +1,50 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <!-- Pass 'ExcludeFromDotnetBuild=false' when building: test projects are excluded by default -->
```

Comment:
Please set `<DisableArcadeExcludeFromBuildSupport>true</DisableArcadeExcludeFromBuildSupport>` in the root's Directory.Build.props instead. We don't need or want the ExcludeFromBuild infrastructure in the VMR orchestrator infrastructure.

You can then remove this comment and the ExcludeFromDotNetBuild properties that you had to add.